src/test/java/uk/gov/hmcts/reform/ccd/documentam/service/impl/DocumentManagementServiceImplTest.java

```diff
@@ -604,6 +604,53 @@ void uploadDocuments_HappyPath() {
         verify(documentStoreClient).uploadDocuments(uploadRequest);
     }
 
+    @Test
+    void uploadDocuments_WithEmptyContent() {
+        final Document document = Document.builder()
+            .size(0L)
+            .mimeType(MIME_TYPE)
+            .originalDocumentName(ORIGINAL_DOCUMENT_NAME)
+            .classification(Classification.PUBLIC)
+            .links(TestFixture.getLinks())
+            .build();
+
+        final DmUploadResponse dmUploadResponse = DmUploadResponse.builder()
+            .embedded(DmUploadResponse.Embedded.builder().documents(List.of(document)).build())
+            .build();
+
+        stubGetSalt();
+        doReturn(dmUploadResponse).when(documentStoreClient).uploadDocuments(any(DocumentUploadRequest.class));
```

Comment:
from service class perspective this test doesn't differ with files with some content. I guess this will be a duplicate - so no need.

Reply:
fixed in [381408c](https://github.com/hmcts/ccd-case-document-am-api/pull/477/commits/381408c3eb8d020a1d13e587142a9ce483cae340)